### PR TITLE
passing orex mapcam tests 

### PR DESCRIPTION
**Pull request by Kelvinrr**
## Description

The original mapcam test was failing after removing the predicted kernels as it used an old image that cannot be spiceinited with reconstructed kernels. The new image has the same name so this mostly removes the kernel parameters fot spiceinit forcing older kernels and changes campt calls to line/samples  with actual target intersections. 

## Related Issue

Part of #4475

## How Has This Been Tested?

All tests are passing locally. 

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read and agree to abide by the [Code of Conduct](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/Code-Of-Conduct.md)
- [x] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] I have added myself to the [.zenodo.json](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/.zenodo.json) document.
- [ ] I have added any user impacting changes to the [CHANGELOG.md](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CHANGELOG.md) document.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [x] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

File: isis/src/osirisrex/tsts/mapcam/Makefile [hunk 1]
```diff
@@ -18,18 +18,12 @@ commands:
 	ocams2isis $(TSTARGS)                                                                        \
 	           from=$(INPUT)/20190303T100344S990_map_iofL2pan_V001.fits                          \
 	           to=$(OUTPUT)/20190303T100344S990_map_iofL2pan_V001-spiceinit.cub                  \
-	           >& /dev/null;
+	           > /dev/null;
```
Comment (Kelvinrr) [inline isis/src/osirisrex/tsts/mapcam/Makefile:21]: I removed the ampersands to allow errors to print out for debugging purposes. Seems like leaving the ampersands is actually better as we can get a better idea as to why something failed in Jenkins logs? 

If anyone disagrees or there is another reason to re-route stderr to /dev/null, I can put them back in.

**Review by jessemapel — APPROVED** [review]
I can't think of any reason to redirect errors on makefile tests. Is there a reason why we aren't specifying the SCLK and PCK anymore?

**Comment by Kelvinrr** [comment]
@jessemapel They are both just very old. That SCLK shouldn't be used anymore in favor of `orx_sclkscet_?????.tsc` SCLKs. The Bennu PCK's latest version is like 16 and is picked up automatically with the kernel selection.

**Comment by Kelvinrr** [comment]
Phase stddev was off on linux, adding `PhaseStandardDeviation = 1e-10` to the tolerances got the tests passing locally so I'll re-run the Jenkins test.

**Pull request merged**